modules/ASB/zones/Arrapago_Reef/npcs/qm6.lua

```diff
@@ -0,0 +1,14 @@
+require('modules/module_utils')
+
+local m = Module:new('qm6')
```

Comment:
Need to make the module name unique in case we get a qm6 in another zone we need to modularize. `arrapago_reef_qm6` is what I would use.